tracer/test/Datadog.Trace.Tests/Telemetry/Collectors/ConfigurationTelemetryCollectorTests.cs

```diff
@@ -180,15 +179,14 @@ public void ConfigurationDataShouldMarkAsManagedOnlyWhenProfilerNotAttached()
 
     [Theory]
     [MemberData(nameof(GetPropagatorConfigurations))]
-    public void ConfigurationDataShouldIncludeExpectedPropagationValues(string propagationStyleExtract, string propagationStyleInject, string propagationStyle, string activityListenerEnabled)
+    public void ConfigurationDataShouldIncludeExpectedPropagationValues(string propagationStyleExtract, string propagationStyleInject, string propagationStyle)
     {
         var collector = new ConfigurationTelemetry();
         var config = new NameValueCollection
         {
             { ConfigurationKeys.PropagationStyleExtract, propagationStyleExtract },
             { ConfigurationKeys.PropagationStyleInject, propagationStyleInject },
             { ConfigurationKeys.PropagationStyle, propagationStyle },
-            { ConfigurationKeys.FeatureFlags.OpenTelemetryEnabled, activityListenerEnabled },
```

Comment:
I think you could keep this line, which helps assert that regardless if it's set or not, the assert is the same